remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerImpl.java

```diff
@@ -44,6 +47,7 @@
 @MBean(objectName = ProtobufMetadataManagerConstants.OBJECT_NAME,
       description = "Component that acts as a manager and container for Protocol Buffers message type definitions in the scope of a CacheManger.")
 public final class ProtobufMetadataManagerImpl implements ProtobufMetadataManager {
+   public static final String SCHEMA_MANAGER_ROLE = "___schema_manager";
```

Comment:
Whouldn't it be nicer to move this constant to the interface rather the impl? Same for ```ScriptingManagerImpl.SCRIPT_MANAGER_ROLE```